deploy/01_deploy.js

```diff
@@ -28,12 +29,10 @@ module.exports = async ({
 
   console.log("----------------------------------------------------");
   console.log('Deploying GovernorAlpha');
-  let quorumVotes = '25000000';
-  let proposalThreshold = '2500000';
-  let votingPeriod = 17280;
+  console.log(ropstenConfig.quorumVotes);
   const governorAlpha = await deploy('GovernorAlpha', {
       from: deployer,
-      args: [timelock.address, btrust.address, foundationInitialAddress, web3.utils.toWei(quorumVotes), web3.utils.toWei(proposalThreshold), votingPeriod],
+      args: [timelock.address, btrust.address, foundationInitialAddress, web3.utils.toWei(ropstenConfig.quorumVotes), web3.utils.toWei(ropstenConfig.proposalThreshold), ropstenConfig.votingPeriod],
```

Comment:
can you select the ropsten config based on the network being deployed to, rather than hardcoded to ropsten?